### DISCUSSION TITLE
Prisma is Awesome

### DISCUSSION THREAD

**Discussion by csenn** [prisma/prisma#9920] [Q&A]
Not sure if this is the right forum, but just wanted to give a quick shout out to the designers behind this lib! I've been using ORMs in different languages for 10 years, and Prisma is just frickin amazing!!! Sure there's always room for improvements, but the APIs, the data structures, the type checking, the docs (for most things), just the theory is awesome. Congrats team!

**Comment by elie222** [comment]
Still needs a lot of work unfortunately :(

Timeouts, poor handling of nested inserts with unique constraints... a whole bunch of issues that need fixing and are super painful... Not sure it comes up in every usecase but when it does it's super frustrating.

**Reply by nikolasburk** [reply]
Hey @elie222  👋  thanks a lot for your feedback! Can you be a bit more precise about the exact issues you're experiencing? Are there already GitHub issues that describe these problems in more detail?

**Comment by nikolasburk** [comment]
Thanks for the shoutout @csenn, really glad to hear that you enjoy using Prisma! 😄 I'll pass your feedback on to our Product and Engineering teams as they're the folks that are taking care of the hard work behind the scenes 🙌

**Reply by felinto-dev** [reply]
you're awesome! 🚀

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
